src/app/(tabs)/story/index.tsx

```diff
@@ -13,17 +14,35 @@ import { RenderHTML } from 'react-native-render-html';
 import { SafeAreaView } from 'react-native-safe-area-context';
 
 import styles from './styles';
-import { storyObject } from '../../../utils/story';
+import { fetchStory } from '../../../queries/stories';
+import { Story } from '../../../queries/types';
 
 function StoryScreen() {
   const [isLoading, setLoading] = useState(true);
   const scrollRef = React.useRef<any>(null);
-  const [story, setStory] = useState<any>();
+  const [story, setStory] = useState<Story>();
+
+  const params = useLocalSearchParams<{ storyId: string }>();
+  const { storyId } = params;
+
+  const scrollUp = () => {
+    scrollRef.current?.scrollTo({ x: 0, y: 0 });
+  };
+
+  useEffect(() => {
+    setLoading(true);
+    (async () => {
+      const storyResponse = await fetchStory(parseInt(storyId as string, 10));
```

Comment:
another workaround i can think of, is that you can check if the storyResponse that is returned is null, and if so you just display "No story found" in large letters in the center of the screen. theoretically, a user should never see this because the story will exist b/c it had to have for us to get to this screen, but it will at least stop the compiler warnings